pkg/sources/adapter/azureservicebussource/adapter_test.go

```diff
@@ -24,18 +24,10 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	servicebus "github.com/Azure/azure-service-bus-go"
-
 	adaptertest "knative.dev/eventing/pkg/adapter/v2/test"
 )
 
 func TestHandleMessage(t *testing.T) {
-	origMsgCompleteFn := messageCompleteFunc
-	messageCompleteFunc = testMessageCompleteFunc
-	t.Cleanup(func() {
-		messageCompleteFunc = origMsgCompleteFn
-	})
```

Comment:
Another good reason IMO to keep the entire processing logic inside `handleMessage()` is that we don't test whether messages are properly "completed" anymore, and that's is super critical.
This test was introduced in response to a bug which I discovered, and we should absolutely keep it to prevent regressions. Un-completed messages lead to messages being redelivered for a while, then dropped.